.github/workflows/ci-sharktank-and-shortfin.yml

```diff
@@ -4,13 +4,14 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-name: Integration Tests
+name: CI - sharktank and shortfin
```

Comment:
I'd call this `CI - shark-platform` and rename the file `ci-shark-platform.yml`

Reply:
Updated to `CI - shark-platform`